### PR TITLE
maplibre-rs monthly

### PR DESCRIPTION
**Pull request by maxammann**

### CODE REVIEW

**Comment by maxammann** [comment]
@wipfli ready for review :)

**Comment by wipfli** [comment]
Do you know why the build is failing, @maxammann?

**Comment by wipfli** [comment]
And do you mind feeding the text to google docs or ms word to do some spell checking?

**Comment by maxammann** [comment]
> And do you mind feeding the text to google docs or ms word to do some spell checking?

I did that already 😅 where do you see typos?

**Comment by maxammann** [comment]
> Do you know why the build is failing, @maxammann?

We are hitting GitHubrat rate limits

**Comment by maxammann** [comment]
@wipfli I fixed some more typos.

File: content/news/2022-12-18-maplibre-rs-monthly/index.md [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+    muted
+    loop
+  ></video>
+  
+  <figcaption>
+    Text Rendering using SDFs
+  </figcaption>
+</figure>
+
+All the above features are not yet merged into the main branch of maplibre-rs as further design changed need to happen.
```
Comment (wipfli) [inline content/news/2022-12-18-maplibre-rs-monthly/index.md:58]:
```suggestion
All the above features are not yet merged into the main branch of maplibre-rs as further design changes need to happen.
```

File: content/news/2022-12-18-maplibre-rs-monthly/index.md [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+
+Apart from those new features, [kim pham](https://github.com/kimpham54) is currently working on parsing the MapLibre style specification, including its expressions.
+Based on this work, the plan is that we generate statistics about the most frequently used style specification features.
+This way we can create issues for all the missing features and prioritize them according to their usage.
+
+Finally, I want to mention that maplibre-rs has now an RFC process described [here](https://maplibre.org/maplibre-rs/docs/book/rfc/0001-rfc-process.html). Based on that, I wrote an RFC about how data is exchanged between WebWorkers and the main thread [here](https://github.com/maplibre/maplibre-rs/pull/223). I think this is a great way to document progress. Let's see how that process will be used in the future!
+
+## 🏠 Housekeeping
+
+The following will summarizes what happened last week on GitHub.
```
Comment (wipfli) [inline content/news/2022-12-18-maplibre-rs-monthly/index.md:71]:
```suggestion
The following will summarize what happened last week on GitHub.
```

File: content/news/2022-12-18-maplibre-rs-monthly/index.md [hunk 3]
```diff
@@ -0,0 +1,143 @@
...
+  Tiles are processed now immediately, instead of distributing work over frames.
+
+- [#226](https://github.com/maplibre/maplibre-rs/pull/226) Add debug lines and improve tile queuing by [@maxammann](https://github.com/maxammann)<br>
+  Red lines around tiles were added for debugging. Queuing of tiles has also been improved.
+
+- [#224](https://github.com/maplibre/maplibre-rs/pull/224) Automatically choose texture format based on adapter by [@maxammann](https://github.com/maxammann)<br>
+  Texture formats are now selected based on what the GPU supports.
+
+- [#222](https://github.com/maplibre/maplibre-rs/pull/222) Improve error handling by [@maxammann](https://github.com/maxammann)<br>
+  Complete rewrite of errors throuout maplibre-rs to follow best practices.
```
Comment (wipfli) [inline content/news/2022-12-18-maplibre-rs-monthly/index.md:85]:
```suggestion
  Complete rewrite of errors throughout maplibre-rs to follow best practices.
```

**Review by wipfli — APPROVED** [review]
Thanks for the nice blog post. You can commit the typos and then merge...

**Comment by maxammann** [comment]
> Thanks for the nice blog post. You can commit the typos and then merge...

I don't have write access to I can not merge :(

**Comment by maxammann** [comment]
The last commits should fix issues because of rate limiting

**Review by wipfli — APPROVED** [review]
Thanks for looking into the rate limiting, looks good to me

**Comment by wipfli** [comment]
https://github.com/maplibre/maplibre.github.io/actions/runs/3750591362/jobs/6370502011#step:6:11 says it hits rate limiting for the charter. Can you maybe also add the github token there?

**Pull request merged**